CHANGELOG.next.asciidoc

```diff
@@ -73,6 +73,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix goroutine leak happening when harvesters are dynamically stopped. {pull}11263[11263]
 - Fix `add_docker_metadata` source matching, using `log.file.path` field now. {pull}11577[11577]
 - Add missing Kubernetes metadata fields to Filebeat CoreDNS module, and fix a documentation error. {pull}11591[11591]
+- LimitReader copies bytes instead of slicing so the underlying big array can be freed. {pull}11524[11524]
```

Comment:
In user facing changelog better mention what has been fixed, not how an implementation detailt has been changed. This helps users to identify if they've been affected.

How about:

`Reduce memory usage if long lines are truncated to fit `max_bytes` limit. The line buffer is copied into a smaller buffer now. This allows the runtime to release unused memory more early.`.